src/cli/Cargo.toml

```diff
@@ -41,10 +41,15 @@ async-trait = "0.1.21"
 ctrlc = "3.1.3"
 futures-lite = { version = "1.7.0" }
 dirs = "1.0.2"
+async-h1 = "2.1.2"
+async-tls = "0.10.0"
```

Comment:
use `fluvio-async-tls` which is our fork to added certificates API

Reply:
You know, I just realized that I'm not even using `async-tls`. I was originally going to, but then I copied the sendgrid client code which is using TLS via `fluvio-future`. Let me know if that works or if I need to change something. I'm going to remove this dependency at least

Reply:
That works